dev/breeze/doc/BREEZE.md

```diff
@@ -93,8 +95,16 @@ Breeze, is not globally accessible until your PATH is updated. Add <USER FOLDER>
 pipx ensurepath
 ```
 
+<<<<<<< HEAD
```

Comment:
Some merge artifacts

Reply:
I got it!  It means I didn't execute pre-commit for this, if I had done this it would catch the errors before pushing them. 

I should remember as a rule, run pre-commit always

Reply:
Oh yeah. Just `pre-commit install` and it will run always when you commit.